examples/CSharp/OpenGL Tutorials/Tutorial 1.4 - Abstractions/Texture.cs

```diff
@@ -1,11 +1,11 @@
-using Silk.NET.OpenGL;
 using System;
-using SixLabors.ImageSharp;
-using SixLabors.ImageSharp.PixelFormats;
+using System.IO;
+using Silk.NET.OpenGL;
+using StbImageSharp;
 
 namespace Tutorial
 {
-    public class Texture : IDisposable
+    public class Texture
```

Comment:
Any reason why you removed `IDisposable`? I see the class still implements the `Dispose` method.

Reply:
My mistake!